Notifier-service/src/interfaces/stream.ts

```diff
@@ -10,5 +10,9 @@ import { Observable } from "rxjs";
  * @interface Stream
  */
 export interface Stream{
-    getStream():Observable<any>
+    getStream():Observable<any>;
+    getInsertStream():Observable<any>;
```

Comment:
* Not necessary here, a postgres stream or mysql stream may have a different interface. The ConferenceStream requires them because it's the interface every other class is using it to use the implementation.